internal/liquids/nova/liquid.go

```diff
@@ -101,12 +196,24 @@ func (l *Logic) ScanCapacity(ctx context.Context, req liquid.ServiceCapacityRequ
 	return liquid.ServiceCapacityReport{}, errors.New("TODO")
 }
 
-// ScanUsage implements the liquidapi.Logic interface.
-func (l *Logic) ScanUsage(ctx context.Context, projectUUID string, req liquid.ServiceUsageRequest, serviceInfo liquid.ServiceInfo) (liquid.ServiceUsageReport, error) {
-	return liquid.ServiceUsageReport{}, errors.New("TODO")
-}
-
 // SetQuota implements the liquidapi.Logic interface.
 func (l *Logic) SetQuota(ctx context.Context, projectUUID string, req liquid.ServiceQuotaRequest, serviceInfo liquid.ServiceInfo) error {
-	return errors.New("TODO")
+	opts := make(novaQuotaUpdateOpts, len(serviceInfo.Resources))
+	for resName := range serviceInfo.Resources {
+		opts[string(resName)] = req.Resources[resName].Quota
+	}
+	return quotasets.Update(ctx, l.NovaV2, projectUUID, opts).Err
+}
+
+func (l *Logic) IgnoreFlavor(flavorName string) bool {
```

Comment:
Present tense in a method name implies an activity, but this method does not cause anyone to ignore anything; it just checks whether something is already ignored. In this case, a past tense feels more appropriate to me:
```suggestion
func (l *Logic) IsIgnoredFlavor(flavorName string) bool {
```